ios/Inji/Images.xcassets/SplashScreen.imageset/Contents.json

```diff
@@ -1,21 +1,23 @@
 {
-  "images": [
+  "images" : [
     {
-      "idiom": "universal",
-      "filename": "image.png",
-      "scale": "1x"
+      "filename" : "splash.png",
+      "idiom" : "universal",
+      "scale" : "1x"
     },
     {
-      "idiom": "universal",
-      "scale": "2x"
+      "filename" : "splash@2x.png",
+      "idiom" : "universal",
+      "scale" : "2x"
     },
     {
-      "idiom": "universal",
-      "scale": "3x"
+      "filename" : "splash@3x.png",
+      "idiom" : "universal",
+      "scale" : "3x"
     }
   ],
-  "info": {
```

Comment:
Is it okay, if we have xcode as author ?

Reply:
Before the image might be added using expo but now modified the splash screen using xcode that's why it is showing author as 'xcode'. There won't be any issue with this and i have confirmed with @dhivya0413 also